### ISSUE TITLE
invoke db_export/db_import will not work as there the container name for postgress is different than expected + more task suggestions

### ISSUE THREAD

**Issue by itheodoridis** [nautobot/nautobot-docker-compose#83]
The name of the postgres service in the docker-compose.postgres.yml is 'db'. This causes the container name to be :
'nautobot-docker-compose-db-1'. The scripts in tasks.py for db_import/db_export reference the container name as 'nautobot-docker-compose-postgres-1' which naturally doesn't exist.
There are several options on how to correct this:

- The service name needs to be corrected ('postgres')
- A container name can be defined in the docker-compose.postgres.yml as an option
- The container name referenced in the script can be changed replacing 'postgres' with 'db'.
- I am sure you can find more

On a side note, task,py could be extended to allow for more docker compose functions to be called with invoke, for example:
```
@task
def logs(context):
    """Show logs for all Nautobot services defined in the docker-compose files."""
    print("Showing Nautobot Logs")
    docker_compose(context, "logs --follow --tail 500")

@task
def softstart(context):
    """Start Nautobot containers when previously stopped (not deleted)."""
    print("Starting Nautobot back up...")
    docker_compose(context, "start")

@task
def softstop(context):
    """Stop Nautobot containers without deleting them."""
    print("Stopping Nautobot Containers...")
    docker_compose(context, "stop")
```

**Comment by jdrew82** [comment]
This has been fixed by #74.

**Comment by jdrew82** [comment]
@itheodoridis If you'd like to submit a PR with the suggested tasks we're happy to consider the additions.
